crates/ln-dlc-node/src/channel.rs

```diff
@@ -62,7 +62,7 @@ impl Channel {
     ) -> Self {
         Channel {
             user_channel_id,
-            channel_state: ChannelState::Pending,
+            channel_state: ChannelState::Announced,
```

Comment:
> 👀 Seems like a bit of a sneaky change. Do we want to introduce separate constructors for Announced and Pending? I worry that there might be a place where we are using new for a different role (coordinator, app, maker) and this change is unwanted.

Well, the initial state of any channel is `Announced` if we upsert a channel which wasn't announced, we created a new one (Announced) and immediately update the channel state there.

> It's quite unfortunate that we're using the same model for app, coordinator and maker, because they all want different things out of the shadow representation of the channel.

Agreed